src/plots/cartesian/layout_attributes.js

```diff
@@ -86,7 +86,7 @@ module.exports = {
         values: ['normal', 'tozero', 'nonnegative'],
         dflt: 'normal',
         role: 'style',
-        editType: 'calc',
+        editType: 'plot',
```

Comment:
It would be nice to add a few test like this [one](https://github.com/plotly/plotly.js/blob/8731cb5d87f305fff8386b5d8eb2f0280752393f/src/traces/scattergl/index.js#L144-L161) that :lock: the new `editType: 'plot'`.

Reply:
... as removing `editType: 'calc'` is a pretty be deal :racehorse: we wouldn't want to regress from that in the future!